src/driver/DriverUtils.ts

```diff
@@ -155,21 +170,28 @@ export class DriverUtils {
             password = usernameAndPassword.substr(firstColon + 1);
         }
 
+        // If replicaSet have value set It as hostlist, If not set like standalone host
         if (replicaSet) {
             hostReplicaSet = hostAndPort;
         } else {
             [host, port] = hostAndPort.split(":");
         }
 
-        return {
+        let connectionUrl: any = {
```

Comment:
The explicit `any` type annotation widens the return type of `parseMongoDBConnectionUrl()` in a regressive manner.

Reply:
I see, But origin of this code before my changed, It's return as `any` too then I try to not change It's type.

But your comment was make sense, I'll try to fixed it's type and test it to make sure that's not effect another soon.